### ISSUE TITLE
About the QST dataset

### ISSUE THREAD

**Issue by hyang0511** [zhangzjn/DTVNet#7]
As described in the paper:

" We split the selected video clips into a training set (containing 1000 clips, totally 244,930 frames), a validation set (containing 100 clips, totally 23,200 frames), and a testing set (containing 67 clips, totally 17,316 frames)."

There are 67 clips for testing and 100 clips for validation. However, the downloaded meta files (test_urls.txt and val_urls.txt) show that there are 100 clips for testing and 67 for validation.

Which version is correct?

**Comment by zhangzjn** [comment]
Follow the first description in the paper, and meta files have been corrected in the [down-link](https://pan.baidu.com/s/1HUmSu-H1ot39ENeesVuz4Q). Thank you!

**Comment by hyang0511** [comment]
Thanks for the clarification.
